src/components/dialogs/create-study-dialog.js

```diff
@@ -137,7 +136,7 @@ const SelectCase = () => {
  * @param {Boolean} open Is the dialog open ?
  * @param {EventListener} onClose Event to close the dialog
  */
-export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
+export const CreateStudyDialog = ({ open, onClose, providedExistingCase }) => {
```

Comment:
Bug fix 
Change  the prop name in the `ContentContextualMenu` component

Reply:
Done